src/main/java/com/snack/news/domain/picks/Pick.java

```diff
@@ -0,0 +1,58 @@
+package com.snack.news.domain.picks;
+
+import com.snack.news.domain.base.BaseTimeEntity;
+import com.snack.news.domain.category.Category;
+import com.snack.news.domain.topic.Topic;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+import javax.persistence.*;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'javax.persistence'](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=4962836)